exporter/stats/stackdriver/stackdriver.go

```diff
@@ -397,3 +399,17 @@ func equalAggWindowTagKeys(md *metricpb.MetricDescriptor, agg stats.Aggregation,
 
 	return nil
 }
+
+func sanitize(s string) string {
+	if len(s) > labelKeySizeLimit {
+		s = s[:labelKeySizeLimit]
+	}
+	return strings.Map(sanitizeRune, s)
+}
+
+func sanitizeRune(r rune) rune {
+	if unicode.IsLetter(r) || unicode.IsDigit(r) || r == 95 {
+		return r
+	}
+	return 95 // Sanitize everything that is not letter, digit or underscore to underscore
```

Comment:
I think we can move this comment to the top of the function to make it
```go
// converts anything that is not a letter or digit to an underscore
func sanitizeRune(r rune) rune {
     if unicode.IsLetter(r) || unicode.IsDigit(r) {
        return r
    }
    // Everything else turns into an underscore
    return '_'
}
```

Reply:
Done.